### PR TITLE
Refresh Browser Section

### PR DESCRIPTION
**Pull request by kimg45**
Update some info in the browser section.

- firefox and chrome are both now on a [four](https://blog.mozilla.org/futurereleases/2019/09/17/moving-firefox-to-a-faster-4-week-release-cycle/) [week](https://chromium.googlesource.com/chromium/src/+/master/docs/process/release_cycle.md) release schedule
- fixed some typos
- Firefox now support fingerprint randomization in the UI
- Swap ublock origin for ublock origin lite using the more secure manifest v3, and chrome lets you control javascript per domain anyway
- add disabling JIT for chrome
- remove kablock since it hasn't been updated in multiple years and there are complaints of it breaking sites bc of outdated blocklists

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -717,15 +717,15 @@ Previous versions of Firefox used a Web Extension SDK that was quite invasive an
...
 
-Create separate Chrome profiles to reduce XSS risk and compartmentalize cookies/identities. In each profile, either disable Javascript in Chrome settings and configure allowed origins - or use [uBlock Origin](https://github.com/gorhill/uBlock) to manage Javascript.
+Create separate Chrome profiles to reduce XSS risk and compartmentalize cookies/identities. In each profile, either disable Javascript in Chrome settings and configure allowed origins. You should also disable the V8 Optimizer for sites where you do use Javascript to further reduce attack surface. You can block trackers with [uBlock Origin Lite](https://chromewebstore.google.com/detail/ublock-origin-lite/ddkjiahejlhfcafbddmgiahcphecmpfh).
```
Comment (drduh) [inline README.md:728]: How is the V8 optimizer disabled? Any further reading on the benefits of doing so?

File: README.md [hunk 2]
```diff
@@ -735,20 +735,18 @@ Read [Chromium Security](https://www.chromium.org/Home/chromium-security) and [C
...
+Safari supports certain unique features that benefit user security and privacy. [Content blockers](https://webkit.org/blog/3476/content-blockers-first-look) enables the creation of content blocking rules without using Javascript. This rule based approach greatly improves memory use, security, and privacy. Safari 11 introduced [Intelligent Tracking Prevention](https://webkit.org/blog/7675/intelligent-tracking-prevention), which removes tracking data stored in Safari after a period of non-interaction by the user from the tracker's website. Safari can randomize your fingerprint to reduce tracking. Safari doesn't support certain features like WebUSB or the Battery API intentionally for security and privacy reasons. Private tabs in Safari have isolated cookies and cache that is destroyed when you close the tab. Safari also support Profiles which are equivalent to Firefox's Multi-Account Containers for separating cookies and browsing. Safari can be made significantly more secure with [lockdown mode](#lockdown-mode), which you can disable per-site. Read more about [tracking prevention](https://webkit.org/tracking-prevention) in Safari.
```
Comment (drduh) [inline README.md:740]: s/you can disable/can be disabled

File: README.md [hunk 3]
```diff
@@ -763,9 +761,9 @@ Web browsers reveal information in several ways, for example through the [Naviga
...
 
-Also be aware of [WebRTC](https://en.wikipedia.org/wiki/WebRTC#Concerns), which may reveal your local or public (if connected to VPN) IP address(es). In Firefox and Chrome/Chromium this can be disabled with extensions such as [uBlock Origin](https://github.com/gorhill/uBlock/wiki/Prevent-WebRTC-from-leaking-local-IP-address) and [rentamob/WebRTC-Leak-Prevent](https://github.com/rentamob/WebRTC-Leak-Prevent). Disabling WebRTC in Safari is only possible with a [system hack](https://github.com/JayBrown/Disable-and-toggle-WebRTC-in-macOS-Safari).
+Also be aware of [WebRTC](https://en.wikipedia.org/wiki/WebRTC#Concerns), which may reveal your local or public (if connected to VPN) IP address(es). In Firefox and Chrome/Chromium this can be disabled with extensions such as [uBlock Origin](https://github.com/gorhill/uBlock/wiki/Prevent-WebRTC-from-leaking-local-IP-address). [Lockdown mode](#lockdown-mode) disables WebRTC in Safari.
```
Comment (drduh) [inline README.md:766]: Can we cite this bit about lockdown mode and WebRTC?

**Review by drduh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kimg45) [reply README.md:728]: There's a [blog post](https://v8.dev/blog/jitless) from V8 and a [blog post](https://microsoftedge.github.io/edgevr/posts/Super-Duper-Secure-Mode/) from the Edge security team about the attack surface of JIT compilers (which is what the V8 optimizer is).

File: README.md (hunk 2 above)
Comment (kimg45) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (kimg45) [reply README.md:766]: I have [this](https://www.sevarg.net/2022/07/20/ios16-lockdown-mode-browser-analysis/) post and my own experience. You can test it out if you want.

File: README.md (hunk 1 above)
Comment (kimg45) [reply README.md]: Added

**Review by drduh — APPROVED** [review]
LGTM. Go ahead and merge your PRs when you're ready. Thank you!

**Comment by kimg45** [comment]
Oh thanks, I don't have write access tho so I can't merge them.

**Pull request merged**